src/pages/Admin/Admin.tsx

```diff
@@ -449,10 +450,13 @@ const Admin: React.FC = () => {
   if (isAdminUserLoading) return <>Loading...</>;
   if (!IS_BURN && !isAdminUser) return <>Forbidden</>;
 
+  if (!user) {
+    return <Redirect to={`/in/${DEFAULT_VENUE}`} />;
```

Comment:
Please use `venueInsideUrl`

Reply:
There is no venueInsideUrl because this is the admin.

Reply:
Nvm, i'll use it.